packages/api/src/external/carequality/organization-template.ts

```diff
@@ -22,82 +25,90 @@ export function buildOrganizationFromTemplate(orgDetails: CQOrgDetails) {
     contactName,
     phone,
     email,
+    role,
+    hostOrgOID,
   } = orgDetails;
 
   const urnOid = "urn:oid:" + oid;
-  const endpointXCPD = getEndpoint(urnOid, urlXCPD, XCPD_STRING);
-  const endpointDQ = urlDQ ? getEndpoint(urnOid, urlDQ, XCA_DQ_STRING) : "";
-  const endpointDR = urlDR ? getEndpoint(urnOid, urlDR, XCA_DR_STRING) : "";
+
+  const endpoints =
+    role === "Implementer"
+      ? getEndpoint(urnOid, XCPD_STRING, urlXCPD) +
+        getEndpoint(urnOid, XCA_DQ_STRING, urlDQ) +
+        getEndpoint(urnOid, XCA_DR_STRING, urlDR)
+      : "";
 
   return `
   <Organization>
```

Comment:
I know this was already done, but can we avoid the new line before the first element? Some XML processors might fail as it happened before.